orderer/common/follower/follower_chain.go

```diff
@@ -186,19 +191,24 @@ func (c *Chain) StatusReport() (types.ClusterRelation, types.Status) {
 
 	clusterRelation := types.ClusterRelationFollower
 	if c.joinBlock != nil {
-		if err := c.amIInChannel(c.joinBlock); err == nil {
+		isMem, _ := c.resources.IsChannelMember(c.joinBlock)
```

Comment:
Just wondering why we don't care about the error here. Is this code path not reachable for solo/kafka orderers?

Reply:
IsChannelMember() is only relevant for cluster-type chains (follower, etcdraft) and is checked against errors when the follower is created. If it errs, the join is rejected since it indicates the join-block is bad. Here, there is virtually no chance of an error, and if it errs, we just treat it as a non member, which does no harm.

Reply:
PS - in the next commit to this PR IsChannelMember will not be implemented by solo & kafka at all.